go.mod

```diff
@@ -2,10 +2,15 @@ module github.com/percona/everest
 
 go 1.22
 
+toolchain go1.22.1
+
+replace github.com/Percona-Lab/percona-version-service => ../percona-version-service
```

Comment:
Waits for https://github.com/Percona-Lab/percona-version-service/pull/197

Reply:
The PR is merged, should we delete the replacement? hopefully at least some of the piplelines will be fixed then😅

Reply:
Updated